ruby/vampires.rb

```diff
@@ -0,0 +1,55 @@
+
+puts "How many employees will be processed?"
+employees = gets.chomp
+
+employees_loop = 0
+
+until employees.to_i == employees_loop
```

Comment:
When doing loop for a specified number of times, look into the Integer class for methods like #times